js/web/lib/wasm/jsep/webnn/webnn.d.ts

```diff
@@ -400,6 +401,9 @@ declare const MLTensorUsage: {
 
 interface MLTensorDescriptor extends MLOperandDescriptor {
   usage: MLTensorUsageFlags;
+  importableToWebGPU?: boolean;
```

Comment:
Should this be `WebGpu`, to stick with typical JS camelCase guidelines? Even Google's JS style guidelines recommend so (e.g. `newCustomerId` over `newCustomerID`, `xmlHttpRequest` over `xmlHTTPRequest`, `supportsIpv6OnIos` over `supportsIpv6OnIOS`). https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined

@a-sully?

Reply:
I'm not blocking on this either way. Resolving for now.